src/inverseRobot.js

```diff
@@ -24,7 +24,20 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const rb1 = Object.keys(robot);
+  const rb2 = Object.values(robot);
```

Comment:
Actually you don't need those to complete the task. There is a simpler way

Try to do it with help of `for...in` loop and  `.hasOwnProperty()` method